docs/dev/run-nb-test.py

```diff
@@ -58,9 +61,14 @@ def main(cli_profile, workspace_tmp_dir, dbfs_init_script_dir, source_dir, nbs):
 
         print(f"Launching runs")
         for nb in nbs:
-            jobs_json['name'] = 'Glow notebook integration test - ' + nb
-            jobs_json['notebook_task'] = {'notebook_path': work_dir + '/' + nb}
-            run_submit = run_cli_cmd(cli_profile, 'runs', ['submit', '--json', json.dumps(jobs_json)])
+            if "hail" in nb:
```

Comment:
It'd be cleaner to have:
```
jobs_json = jobs_json_base
if "hail" in nb:
  jobs_json = hail_jobs_json
jobs_json['notebook_task'] = {'notebook_path': work_dir + '/' + nb}
run_submit = run_cli_cmd(cli_profile, 'runs', ['submit', '--json', json.dumps(jobs_json)])
```

Reply:
that does seem nicer, although I am leaning towards providing an arg to the python script instead, so we do not need `if "x" in nb:`